Assets/MRTK/SDK/Features/UX/Scripts/Pointers/BaseControllerPointer.cs

```diff
@@ -40,6 +40,15 @@ public abstract class BaseControllerPointer : ControllerPoseSynchronizer, IMixed
         [Tooltip("The action that will enable the raise the input event for this pointer.")]
         protected MixedRealityInputAction pointerAction = MixedRealityInputAction.None;
 
+        [SerializeField]
+        [Tooltip("The action that will enable the raise the input drag event for this pointer.")]
+        protected MixedRealityInputAction dragAction = MixedRealityInputAction.None;
```

Comment:
Is it possible to rename this from "dragAction" to "grabAction"? We're considering adding "grab" and "release" voice commands, and for the sake of standardizing the paradigm I think "grab" is a better work than "drag"
https://github.com/microsoft/MixedRealityToolkit-Unity/issues/9039

This will help in the future if we decide that we want a button to turn on a "grabbing" state and a button to exit the "grabbing" state.

Looks good to me otherwise.

Reply:
Sure. A new commit is added.
Thanks for the review!